more/free-programming-cheatsheets.md

```diff
@@ -68,6 +69,11 @@
 * [GitHub Cheat Sheet](https://github.com/tiimgreen/github-cheat-sheet) - Tim Green (Markdown)
 
 
+### Go
```

Comment:
:information_source: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.